src/message_replies/givePoint.js

```diff
@@ -2,65 +2,92 @@ import { getUserIdFromMention } from '../utils/getUserIdFromMention.js';
 import { isUserValid } from '../utils/isUserValid.js';
 import BuddiesModel from '../../models/BuddiesModel.js';
 
-async function givePoint(command, interaction, isMessage) {
-  const mentionId = getUserIdFromMention(command);
+const ARRAY_OF_POINTS = ['1️⃣', '2️⃣', '3️⃣', '4️⃣', '5️⃣', '6️⃣', '7️⃣', '8️⃣', '9️⃣'];
```

Comment:
Perhaps add this to SIMPLE_MODELS? Not sure that it's necessary, as I can't think another place where we would use this array but its a thought.

What are your thoughts?

Reply:
I don't see why this change. this is something very specific to points right now, but i'll keep this in mind